recipes-kernel/linux/linux-smelt/defconfig

```diff
@@ -3531,31 +3464,3 @@ CONFIG_ARCH_HAS_ATOMIC64_DEC_IF_POSITIVE=y
 CONFIG_QMI_ENCDEC=y
 # CONFIG_QMI_ENCDEC_DEBUG is not set
 # CONFIG_VIRTUALIZATION is not set
-CONFIG_IP_NF_MULTIPLE_TABLES=y
-CONFIG_CGROUP_MEM_RES_CTLR=y
-CONFIG_CGROUP_MEM_RES_CTLR_SWAP=y
-CONFIG_CGROUP_MEM_RES_CTLR_KMEM=y
-CONFIG_BLK_SCHED=y
-CONFIG_DEBUG_RODATA=y
-CONFIG_DEVPTS_FS=y
```

Comment:
Doesn't the ramdisk fail without this https://github.com/AsteroidOS/meta-asteroid/blob/master/recipes-core/initrdscripts/initramfs-scripts-android/init.sh#L24 ?

Reply:
Hm, looks like this option doesn't exist...
I believe that the option in this kernel is named `CONFIG_UNIX98_PTYS` which is enabled, see: https://android.googlesource.com/kernel/msm/+/refs/heads/android-msm-smelt-3.10-marshmallow-mr1-wear-release/fs/devpts/Makefile